src/AppAny.Quartz.EntityFrameworkCore.Migrations/QuartzModel.cs

```diff
@@ -4,7 +4,7 @@ namespace AppAny.Quartz.EntityFrameworkCore.Migrations
 {
   public class QuartzModel
   {
-    public string? Prefix { get; set; } = "qrtz_";
+    public string? Prefix { get; set; } = "QRTZ_";
```

Comment:
This prefix should not be configured here. Each provider should decide a prefix to use

With this change PostgreSQL provider generates broken migrations with upper case prefix
Without this change seems like SqlServer + Sqlite...

I think that this default prefix should be placed inside each `QuartzModelBuilder` extension